compiler/generator/instructions_compiler_jax.cpp

```diff
@@ -0,0 +1,178 @@
+/************************************************************************
+ ************************************************************************
+ FAUST compiler
+ Copyright (C) 2017-2021 GRAME, Centre National de Creation Musicale
+ ---------------------------------------------------------------------
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU Lesser General Public License as published by
+ the Free Software Foundation; either version 2.1 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public License
+ along with this program; if not, write to the Free Software
+ Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ ************************************************************************
+ ************************************************************************/
+
+#include "instructions_compiler_jax.hh"
+#include "ensure.hh"
+#include "ppsig.hh"
+#include "sigtyperules.hh"
+
```

Comment:
Be sure to check that all methods here are necessary, that is are **really** JAX backend specific.

Reply:
Yes I’ve checked again.